app/models/profile.rb

```diff
@@ -52,6 +52,10 @@ def self.verified
     where.not(verified_at: nil)
   end
 
+  def self.facial_match
+    where(idv_level: FACIAL_MATCH_IDV_LEVELS)
+  end
+
```

Comment:
will probably be a helpful 'scope'

Reply:
this is the one we're having a convo about in Slack right? We are going to change this?

Reply:
yes, I just caught up with this. Thanks